config/src/config/mod.rs

```diff
@@ -355,12 +355,10 @@ impl NodeConfig {
             let validator_network = self.validator_network.as_mut().unwrap();
             validator_network.random_with_peer_id(rng, Some(peer_id));
             // We want to produce this key twice
-            let mut cloned_rng = rng.clone();
             test.random_execution_key(rng);
 
             let mut safety_rules_test_config = SafetyRulesTestConfig::new(peer_id);
             safety_rules_test_config.random_consensus_key(rng);
-            safety_rules_test_config.random_execution_key(&mut cloned_rng);
```

Comment:
I don't understand the implications of removing this here. Are you sure it's safe?

Reply:
I see this change propagates further down in other files: e.g., `config/src/config/safety_rules_config.rs`

Reply:
all the execution key related changes are just cleanup